src/WebJobs.Script.WebHost/Web.config

```diff
@@ -43,7 +43,7 @@
       <remove name="ExtensionlessUrlHandler-Integrated-4.0" />
       <remove name="OPTIONSVerbHandler" />
       <remove name="TRACEVerbHandler" />
-      <add name="ExtensionlessUrlHandler-Integrated-4.0" path="*." verb="*" type="System.Web.Handlers.TransferRequestHandler" preCondition="integratedMode,runtimeVersionv4.0" />

```

Comment:
Will send this as a seperat CR.

---
In reply to: [132803115](https://github.com/Azure/azure-webjobs-sdk-script/pull/1749#discussion_r132803115) [](ancestors = 132803115)